initializr-web/src/main/resources/templates/home.mustache

```diff
@@ -18,23 +18,30 @@ new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
 <body>
 <form id="form" action="/starter.zip" method="get" role="form">
     <nav class="navbar header-nav" role="navigation" aria-label="main navigation">
-        <div class="container">
+        <div class="small-container container">
             <div class="navbar-brand">
                 <a class="navbar-item azure-logo">
                     <img src="{{#linkTo}}/img/azure_logo.png{{/linkTo}}" alt="Microsoft Azure">
                 </a>
             </div>
-            <div class="navbar-menu">
+            <div class="navbar-menu is-active">
                 <div class="navbar-start">
                     <h1 class="navbar-item title has-text-white" >
                         Azure Spring Cloud Playground
                     </h1>
                 </div>
+                <div class="navbar-end" id="login_link">
```

Comment:
Can we also add sth. like below?
<a href="https://azure.microsoft.com/en-us/free/" id="mobile-navigation-free-account" class="header-button-primary" title="Free account" data-event="global-clicked-primarycta">Free account</a>

Link: https://azure.microsoft.com/en-us/

Reply:
OK

Reply:
Free account link added.